homeassistant/components/bangolufsen/const.py

```diff
@@ -0,0 +1,309 @@
+"""Constants for the Bang & Olufsen integration."""
+
+from __future__ import annotations
+
+from enum import Enum, StrEnum
+import logging
+from typing import Final, cast
+
+from mozart_api.models import (
+    PlaybackContentMetadata,
+    PlaybackProgress,
+    RenderingState,
+    Source,
+    SourceArray,
+    SourceTypeEnum,
+    VolumeLevel,
+    VolumeMute,
+    VolumeState,
+)
+from mozart_api.mozart_client import MozartClient
+
+from homeassistant.components.media_player import MediaPlayerState, MediaType
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.device_registry import DeviceEntry
+
+
+class ART_SIZE_ENUM(Enum):
+    """Enum used for sorting images that have size defined by a string."""
+
+    small = 1
+    medium = 2
+    large = 3
```

Comment:
Definitions like these don't seem relevant for the Home Assistant integration, and thus should be in a supporting API, not in Home Assistant. How come they're not in `mozart_api`?

Reply:
It seemed like clutter for the API client, but I guess it is also clutter for the Home Assistant integration. I've moved this functionality to `mozart_api`.